lib/school.rb

```diff
@@ -0,0 +1,23 @@
+class School
+
+attr_reader :start_time, :hours_in_school_day, :student_names
+
+  def initialize (start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = student_names
```

Comment:
This should be set to an empty array.

Reply:
@mikedao Just to be clear, you're talking about the `@student_names` instance variable?